app/src/main/java/com/stacker/simplegithubapp/Follower.kt

```diff
@@ -0,0 +1,51 @@
+package com.stacker.simplegithubapp
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+
+class Follower : Fragment() {
```

Comment:
```suggestion
class FollowerFragment : Fragment() {
```
이렇게 프래그먼트는 프래그먼트라고 써줘야됨